aws-android-sdk-s3/src/main/java/com/amazonaws/mobileconnectors/s3/transferutility/DownloadTask.java

```diff
@@ -144,8 +144,6 @@ public Boolean call() {
                      */
                     updater.updateState(download.id, TransferState.WAITING_FOR_NETWORK);
                     LOGGER.debug("Network Connection Interrupted: " + "Moving the TransferState to WAITING_FOR_NETWORK");
-                    ProgressEvent resetEvent = new ProgressEvent(0);
-                    resetEvent.setEventCode(ProgressEvent.RESET_EVENT_CODE);
```

Comment:
Shouldn't we be resetting the progress upon loss of network and then pass `reserEvent` to the `progressListener`?